### PR TITLE
Drop environment customization for xml

### PR DESCRIPTION
**Pull request by nik9000**
Since we don't depend on docbook we don't need to set up the environment
for xml any more.

### CODE REVIEW

File: build_docs.pl [hunk 1]
```diff
@@ -726,12 +726,7 @@ sub push_changes {
 #===================================
 sub init_env {
 #===================================
-    $ENV{SGML_CATALOG_FILES} = $ENV{XML_CATALOG_FILES} = join ' ',
-        file('resources/docbook-xsl-1.78.1/catalog.xml')->absolute,
-        file('resources/docbook-xml-4.5/catalog.xml')->absolute;
-
     $ENV{PATH}
-        = dir('resources/asciidoc-8.6.8/')->absolute
         . ":$FindBin::RealBin:"
```
Comment (olksdr) [inline build_docs.pl]: should be a `=` sign here maybe?

Comment (nik9000) [reply build_docs.pl]: Yeah! I wonder why the tests passed.....

Comment (nik9000) [reply build_docs.pl]: I've entirely dropped the path customizations. I guess we didn't need them!

**Comment by nik9000** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

Just for a little extra paranoia.

**Comment by nik9000** [comment]
Dropping the path customization did the trick.

**Pull request merged**